.github/workflows/clam-av-scan.yml

```
@@ -0,0 +1,182 @@
+name: av-clamav
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Release tag to scan (e.g., v0.15.16)'
+        required: false
+  release:
+    types: [published]
+  pull_request:
+  pull_request:
```

actionlint (1.7.8)

key "pull_request" is duplicated in "on" section. previously defined at line:10,col:3

(syntax-check)